### PR TITLE
fix meta file typo

### PR DESCRIPTION
**Pull request by Lin-Youth**

### CODE REVIEW

File: easynlp/core/trainer.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def resume_from_ckpt(self, model_module, args):
         model_file = args.resume_from_checkpoint + '.bin'
         if 'oss://' in args.resume_from_checkpoint:
             local_file = 'easynlp_resume_pytorch_model.meta.bin'
-            io.download(model_file, local_file)
+            io.download(meta_file, local_file)
```
Comment (minghui) [inline easynlp/core/trainer.py:118]: meta_file not defined?

Comment (Lin-Youth) [reply easynlp/core/trainer.py:118]: should be "meta_file" here, but "model_file" was written twice : )

Comment (minghui) [reply easynlp/core/trainer.py:118]: nice catch~

**Review by minghui — APPROVED** [review]
lgtm

**Pull request merged**